### PR TITLE
Add option to disable lures and show raid count on homepage

### PR DESCRIPTION
**Pull request by 123FLO321**
## Description
This PR adds an option to remove all lures visually and adds an option to show active raid count on home page.

## Motivation and Context
Preparation for upcoming Monocle PR

## How Has This Been Tested?
Local instance and working fine.

## Screenshots (if appropriate):
raids

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by michikrug** [comment]
It's not a good idea to hide lures and show raids when setting an option called "no_lures".
It's okay to hide them, but please separate the raids stuff.

**Comment by 123FLO321** [comment]
@michikrug Separated no_lures and homepage_raids

File: core/process/aru.php [hunk 1]
```diff
@@ -115,14 +124,14 @@
 		$data = $result->fetch_object();
 
 		// Yellow
-		$values[] = $data->total;
+		//$values[] = $data->total;
```
Comment (michikrug) [inline core/process/aru.php]: ?

File: core/process/aru.php [hunk 2]
```diff
@@ -115,14 +124,14 @@
...
 
 		$req = "SELECT COUNT(DISTINCT(gym_id)) AS total FROM gym WHERE team_id = '0'";
 		$result = $mysqli->query($req);
 		$data = $result->fetch_object();
 
 		// Neutral
-		$values[] = $data->total;
+		//$values[] = $data->total;
```
Comment (michikrug) [inline core/process/aru.php]: ?

File: core/process/data.loader.php [hunk 3]
```diff
@@ -393,36 +393,6 @@
 else {
 	$home = new stdClass();
 
-	// Right now
```
Comment (michikrug) [inline core/process/data.loader.php]: why did you remove this part?

**Review by michikrug — CHANGES_REQUESTED** [review]

File: core/process/aru.php (hunk 1 above)
Comment (123FLO321) [reply core/process/aru.php]: That was on accident will revert that

File: core/process/data.loader.php (hunk 3 above)
Comment (123FLO321) [reply core/process/data.loader.php]: As far as i can tell this is unused right?

Comment (michikrug) [reply core/process/data.loader.php]: Okay. It seems so. 👍

**Comment by javisabalete** [comment]
I'll try this PR tomorrow 👍

**Comment by darkelement1987** [comment]
Works as intended!

**Review by darkelement1987 — APPROVED** [review]

**Comment by Obihoernchen** [comment]
@michikrug all fine now?

**Comment by javisabalete** [comment]
works fine

**Comment by michikrug** [comment]
@Obihoernchen not tested yet

**Comment by Obihoernchen** [comment]
looks fine to me. One more tester and I'll merge.

**Review by Obihoernchen — APPROVED** [review]
